src/test/ui/consts/const-err2.rs

```diff
@@ -5,6 +5,8 @@
 
 #![feature(rustc_attrs)]
 #![allow(exceeding_bitshifts)]
+// compile-flags: -C overflow-checks=on -O
```

Comment:
Oh sorry. That wasn't clear. Please add a *new* test with that flag. You can copy an existing test and add it to that, but don't modify existing tests. We need to test both with and without the flag

Reply:
Ah, okay, I'll do that.

Reply:
Fixed tests again